tumblebit/config.py

```diff
@@ -0,0 +1,3 @@
+LIBRESSL_PATH = "/usr/local/lib/libssl.so"
```

Comment:
This should be commented out. The user should uncomment and change to the correct path if the default paths don't work.  Will mention this in the README.